### ISSUE TITLE
Cannot install UI. Get key error. Same as two closed issues. 

### ISSUE THREAD

**Issue by Traphowze** [RackunSec/Summon#7]
Installing summon UI fails. /etc/demon/summon.conf is empty. How do I Regen the file as it's supposed to be?

**Comment by RackunSec** [comment]
Currently there is no code that checks for an empty file, since I assumed ConfigParser was working correctly - as it did in all of my testing. That file should never be empty unless your version of ConfigParser is broken. You can add it to the file manually or you should be able to delete the file and re-run Summon to see if it can make the file properly after asking you if the path is correct.

**Comment by RackunSec** [comment]
Here is the method that is called each time Summon is ran:
```python    
def check_config_file(self):
        if not os.path.exists("/etc/demon/summon.conf"):
            ans=input(f"{self.style.ques} We are running from {self.style.RED}{self.cwd}{self.style.RST} Would you like to make this your Summon binary path [y/n]? ")
            if ans=="y":
                config=ConfigParser()
                config['SUMMON']= {"summon_path":self.cwd}
            with open("/etc/demon/summon.conf","w") as config_file:
                config.write(config_file)
```
Did Summon ask you the question listed in the code snippet above? Did you get an error when you answered it?

**Comment by Traphowze** [comment]
No. Summon never asked me anything. It spits the same string that the first
issue had listed. Ending with key error SUMMON.
